OneBlink.SDK/models/Form.cs

```diff
@@ -267,5 +276,9 @@ public bool continueWithAutosave
         {
             get; set;
         }
+        public List<string> customCssClasses

```

Comment:
Docs were not updated for this https://github.com/oneblink/sdk-dotnet/blob/master/docs/models/Form.md